.gitattributes

```diff
@@ -0,0 +1,3 @@
+*.m2 linguist-language=Macaulay2
```

Comment:
Do we have to do something else now so that files such as those at https://github.com/Macaulay2/M2/blob/master/M2/Macaulay2/m2/debugging.m2 get highlighted beautifully?

Reply:
The PR to add Macaulay2 highlighting to Linguist has been merged, but Linguist needs to make a release before it goes into effect. That should be soon, but ideally we should update the [language grammar](https://github.com/Macaulay2/language-macaulay2/blob/master/grammars/macaulay2.cson) to include recent changes before then.